redaxo/src/core/lib/sql/sql.php

```diff
@@ -415,14 +415,23 @@ public function setQuery($query, array $params = [], array $options = [])
         }
 
         try {
-            $this->stmt = rex_timer::measure(__METHOD__, static function () use ($pdo, $query) {
-                return $pdo->query($query);
+            $this->stmt = rex_timer::measure(__METHOD__, function () use ($pdo, $query) {
+                error_clear_last();
+
+                if (false !== $stmt = @$pdo->query($query)) {
```

Comment:
```suggestion
                // since we are in Exception-Mode, PDO should throw in case of errors.
                // it seems there are rare cases where it still returns false, which we try to handle here
                if (false !== $stmt = @$pdo->query($query)) {
```